src/OverlayTrigger.js

```diff
@@ -281,19 +281,21 @@ const OverlayTrigger = React.createClass({
 
   _getContainerDimensions() {
     const containerNode = this.getContainerDOMNode();
-    let width, height;
+    let width, height, scroll;
+
     if (containerNode.tagName === 'BODY') {
       width = window.innerWidth;
       height = window.innerHeight;
+      scroll =
+        domUtils.ownerDocument(containerNode).documentElement.scrollTop ||
```

Comment:
I need specifically `ownerDocument` there for consistency with other positioning calculations. While there is some duplicated effort there, I think it's cleaner to do everything off the `containerNode` element than to e.g. duplicate that branch in `getContainerDOMNode`.